frontend/src/modules/member/components/list/member-list-toolbar.vue

```diff
@@ -216,6 +228,16 @@ export default {
       }
     },
 
+    async handleMergeMembers() {
+      const [firstMember, secondMember] = this.selectedRows
+      await MemberService.merge(firstMember, secondMember)
+
+      Message.success('Members merged successfuly')
```

Comment:
Can you also add an error Message for this? In staging I'm getting 500 after I try to merge them and I don't get feedback of any error